### YamlMime:ManagedReference
items:
- uid: System.Runtime.Remoting.ServerException
  id: ServerException
  children:
  - System.Runtime.Remoting.ServerException.#ctor
  - System.Runtime.Remoting.ServerException.#ctor(System.String)
  - System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: ServerException
  nameWithType: ServerException
  fullName: System.Runtime.Remoting.ServerException
  type: Class
  summary: "當用戶端連接至無法擲回例外狀況的非.NET Framework 應用程式時，用戶端通訊錯誤擲回的例外狀況。"
  remarks: "ServerException 會使用 HRESULT COR_E_SERVER，0x8013150E 的值。       ServerException 會使用預設<xref:System.Object.Equals%2A>支援參考相等的實作。</xref:System.Object.Equals%2A>       如需 ServerException 的執行個體的初始屬性值的清單，請參閱 ServerException 建構函式。"
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public class ServerException : SystemException
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Runtime.Remoting.ServerException.#ctor
  id: '#ctor'
  parent: System.Runtime.Remoting.ServerException
  langs:
  - csharp
  name: ServerException()
  nameWithType: ServerException.ServerException()
  fullName: System.Runtime.Remoting.ServerException.ServerException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "初始化的新執行個體<xref href=&quot;System.Runtime.Remoting.ServerException&quot;></xref>使用預設屬性的類別。"
  remarks: "<xref:System.Runtime.Remoting.ServerException>繼承自<xref:System.SystemException>類別。</xref:System.SystemException></xref:System.Runtime.Remoting.ServerException> 下表顯示<xref:System.Runtime.Remoting.ServerException>類別的執行個體的初始屬性值: |屬性 |值 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|`null`|  |<xref:System.Exception.Message%2A>|空字串 (&quot;&quot;) |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> </xref:System.Runtime.Remoting.ServerException>"
  syntax:
    content: public ServerException ();
    parameters: []
  overload: System.Runtime.Remoting.ServerException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.Remoting.ServerException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Runtime.Remoting.ServerException
  langs:
  - csharp
  name: ServerException(String)
  nameWithType: ServerException.ServerException(String)
  fullName: System.Runtime.Remoting.ServerException.ServerException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "初始化的新執行個體<xref href=&quot;System.Runtime.Remoting.ServerException&quot;></xref>類別具有指定的訊息。"
  remarks: "<xref:System.Runtime.Remoting.ServerException>繼承自<xref:System.SystemException>類別。</xref:System.SystemException></xref:System.Runtime.Remoting.ServerException> 下表顯示<xref:System.Runtime.Remoting.ServerException>類別的執行個體的初始屬性值: |屬性類型 |條件 |  |-------------------|---------------|  |<xref:System.Exception.InnerException%2A>|`null`|  |<xref:System.Exception.Message%2A>|`message`字串 |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> </xref:System.Runtime.Remoting.ServerException>"
  syntax:
    content: public ServerException (string message);
    parameters:
    - id: message
      type: System.String
      description: "描述例外狀況的訊息"
  overload: System.Runtime.Remoting.ServerException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Runtime.Remoting.ServerException
  langs:
  - csharp
  name: ServerException(String,Exception)
  nameWithType: ServerException.ServerException(String,Exception)
  fullName: System.Runtime.Remoting.ServerException.ServerException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Runtime.Remoting
  summary: "初始化的新執行個體<xref href=&quot;System.Runtime.Remoting.ServerException&quot;></xref>類別指定的錯誤訊息以及造成此例外狀況的內部例外狀況的參考。"
  remarks: "做為前一個例外狀況直接結果應該包括在前一個例外狀況的參考就會擲回的例外狀況<xref:System.Exception.InnerException%2A>屬性。</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>屬性會傳回相同的值傳遞到建構函式，或`null`如果<xref:System.Exception.InnerException%2A>屬性沒有提供內部例外值給建構函式。</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       下表顯示<xref:System.Runtime.Remoting.ServerException>類別的執行個體的初始屬性值: |屬性 |值 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|內部例外狀況參考 |  |<xref:System.Exception.Message%2A>|錯誤訊息字串 |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A> </xref:System.Runtime.Remoting.ServerException>"
  syntax:
    content: public ServerException (string message, Exception InnerException);
    parameters:
    - id: message
      type: System.String
      description: "解釋例外狀況原因的錯誤訊息。"
    - id: InnerException
      type: System.Exception
      description: "造成目前例外狀況的例外狀況。 如果`InnerException`參數不是<xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;></xref>中, 引發目前的例外狀況<xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;></xref>處理內部例外狀況區塊。"
  overload: System.Runtime.Remoting.ServerException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.Runtime.Remoting.ServerException.#ctor
  parent: System.Runtime.Remoting.ServerException
  isExternal: false
  name: ServerException()
  nameWithType: ServerException.ServerException()
  fullName: System.Runtime.Remoting.ServerException.ServerException()
- uid: System.Runtime.Remoting.ServerException.#ctor(System.String)
  parent: System.Runtime.Remoting.ServerException
  isExternal: false
  name: ServerException(String)
  nameWithType: ServerException.ServerException(String)
  fullName: System.Runtime.Remoting.ServerException.ServerException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)
  parent: System.Runtime.Remoting.ServerException
  isExternal: false
  name: ServerException(String,Exception)
  nameWithType: ServerException.ServerException(String,Exception)
  fullName: System.Runtime.Remoting.ServerException.ServerException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Runtime.Remoting.ServerException.#ctor*
  parent: System.Runtime.Remoting.ServerException
  isExternal: false
  name: ServerException
  nameWithType: ServerException.ServerException
